src/main/java/Command.java

```diff
@@ -0,0 +1,107 @@
+public class Command {
+    private Boolean isExit;
+    private String command;
+    private String arguments;
+    private String description;
+    private String by;
+    private String from;
+    private String to;
+
+    public Command() {
+        this.isExit = false;
+    }
+
+    public void exit() {
+        this.isExit = true;
+    }
+
+    public Boolean getIsExit() {
+        return this.isExit;
+    }
+
+    /**
+     * Splits user input into command and arguments.
+     *
+     * @param input User input.
+     */
+    public void parseCommand(String input) {
+        String[] parts = input.split(" ", 2);
```

Comment:
Perhaps you could change "2" into a variable name that is relevant such as "twoSubString"